### PR TITLE
add time-slot and facility entites

### PR DESCRIPTION
**Pull request by Muhammed9955**
Please add review to close this pull request

### CODE REVIEW

**Review by AmrEzzEldeen — APPROVED** [review]
Reviewed

File: src/main/java/com/teamc/fms/entity/Facility.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Facility {
+    @Id
+    private String id;
+    @NotNull
+    private String name;
+    @NotNull
```
Comment (Mahmoud-Shosha) [inline src/main/java/com/teamc/fms/entity/Facility.java]: desciption is nullable
image

**Review by Mahmoud-Shosha — CHANGES_REQUESTED** [review]

File: src/main/java/com/teamc/fms/entity/Facility.java (hunk 1 above)
Comment (Muhammed9955) [reply src/main/java/com/teamc/fms/entity/Facility.java]: Okay I'm gonna chnage it

**Review by Mahmoud-Shosha — APPROVED** [review]
approved

**Pull request merged**